### ISSUE TITLE
introduce log levels

### ISSUE THREAD

**Issue by converge** [kubefirst/kubefirst#756]
In order to provide better logs for the users, as a Kubefirst CLI user I want to have logs that are more concise, and distinguish warnings, error, debug and info.

Acceptance criteria:

AC1: it doesn't impact current standard log library
AC2: current standard library and zero log works well together
AC3: it uses zero log library
AC4: the update logs provide levels
AC5: default level is set to debug (all logs levels will be displayed)
AC6: there is a flag to change the log level (e.x: --log-level=info)

There is a initial exploration[1] how to achieve this using Go zero library.

[1] https://github.com/kubefirst/kubefirst/pull/164

**Comment by 6za** [comment]
We should be aware the most of the k8s operator will be based on `go.uber.org/zap v1.23.0` as this seems to be the logger added by the code generator for operators. 

Reference: https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/


We we are shopping for logging tools, we should weight that on the choices.

cc: @jarededwards  / @johndietz 

I didn't researched if it is possible to change and so.. but I think to save tech-debt maybe we want to play ball with eco-system and follow along.  I have no personal preference for it, beyond it seems to be the one operators use.

**Comment by converge** [comment]
stak considered:

Logrus → no new implementations / maintenance-mode / dead

Zap → requires extra libraries to provide nice UX colors

Zerolog → provide colors without extra libs.

Also considered Logr interface (logging abstraction layer), it accepts zerolog(zerologr), but adds unnecessary complexibility.

Logr example:

```go
  zerolog.SetGlobalLevel(zerolog.DebugLevel)
	zl := zerolog.New(os.Stdout)
	log := zerologr.New(&zl)
	log = log.WithName("MyName")
	log = log.WithValues("module", "example")

	log.Info("hello", "val1", 1, "val2", map[string]int{"k": 1})
	log.V(1).Info("you should see this")
	log.V(1).V(1).Info("you should NOT see this")
	log.Error(E{"an error occurred"}, "goodbye", "code", -1)
	
```

Zerolog:

```go
func main() {
    zerolog.TimeFieldFormat = zerolog.TimeFormatUnix

    log.Info().Msg("hello world")
    log.Warn().Msg("hello world")
    log.Debug().Msg("hello world")
    log.Error().Err(err).Msg("hello world")

}
```

**Comment by converge** [comment]
implemented for 1.11
